app/main.py

```diff
@@ -2,0 +1,16 @@
+from app.cafe import Cafe
+from app.errors import NotVaccinatedError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for visitor in friends:
+        try:
+            cafe.visit_cafe(visitor)
+        except NotWearingMaskError:
+            masks_to_buy += 1
+        except NotVaccinatedError:
+            return "All friends should be vaccinated"
```

Comment:
Instead of catching `NotVaccinatedError`, catch `VaccineError` to handle both `NotVaccinatedError` and `OutdatedVaccineError` in a single `except` clause. This aligns with the task requirements for comprehensive error handling.